AzureMonitorAgent/ama_tst/modules/error_codes.py

```diff
@@ -1,18 +1,12 @@
 # # General Errors
 NO_ERROR = 0
 USER_EXIT = 1
-# ERR_SUDO_PERMS = 100
+ERR_SUDO_PERMS = 100
```

Comment:
I mentioned above, but you're right that this error code isn't needed - feel free to remove it (and any references to it) completely from the code